examples/AeroSharp.Examples.HelloWorld/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Threading.Tasks;
+using AeroSharp.Connection;
+using AeroSharp.DataAccess;
+
+namespace AeroSharp.Examples.HelloWorld
+{
+    /// <summary>
+    /// The simplest example of connecting to a local instance of Aerospike and writing/reading some data.
+    /// 
+    /// Before running this, run `start_aerospike_in_docker.sh` to start a preconfigured local Aerospike instance.
```

Comment:
That shell script currently lives in the integration tests directory. Given that it's also in use here, should we consider moving it to a `scripts` or `utilities` directory? Either way, the reference should probably be updated so people know where to find said script.